endoreg_db/management/commands/init_default_ai_model.py

```
@@ -0,0 +1,101 @@
+Django management command to initialize the default AI model with metadata.
+This command ensures that a default AI model exists with proper ModelMeta records.
+"""
+
+from django.core.management.base import BaseCommand
+from django.core.management import call_command
+from pathlib import Path
+from endoreg_db.data.ai_model_meta import (
+    default_multilabel_classification
+)
+
+from endoreg_db.models import AiModel, ModelMeta
```

Ruff (0.11.9)

`endoreg_db.models.ModelMeta` imported but unused

Remove unused import: `endoreg_db.models.ModelMeta`

(F401)